resources/agents.json

```
@@ -0,0 +1,367 @@
+{
```

Please add a `$schema`.
```suggestion
  "$schema": "./agents.schema.json",
```